includes/class-stats.php

```diff
@@ -930,7 +930,7 @@ public function get_order_item_earnings( $query = array() ) {
 		$this->query_vars['table']             = $this->get_db()->edd_order_items;
 		$this->query_vars['column']            = true === $this->query_vars['exclude_taxes'] ? 'total - tax' : 'total';
 		$this->query_vars['date_query_column'] = 'date_created';
-		$this->query_vars['status']            = array( 'complete' );
+		$this->query_vars['status']            = array( 'complete', 'refunded', 'partially_refunded' );
```

Comment:
@robincornett These need to account for renewals somehow as well.

Reply:
Since this is querying the order items table, renewals are included in this--the order item status for renewal orders is `complete`. However, the inner join for `get_order_item_count` does need to be updated.